torch/_inductor/ir.py

```diff
@@ -4278,6 +4302,7 @@ def __init__(
                 self.kernel = (
                     f"{kernel.__module__.replace('._ops.', '.ops.')}.{kernel.__name__}"
                 )
+        self.abi_compatible_kernel = None
```

Comment:
not sure why the typechecker didn't complain, but shouldn't this line be in the constructor of `ExternKernelAlloc`, since `ExternKernelAlloc.codegen` references it?

Reply:
Good catch! Fixed. Thanks.